apps/coscrad-frontend/src/components/higher-order-components/categorizable-page-layout.tsx

```diff
@@ -0,0 +1,174 @@
+import { AggregateCompositeIdentifier } from '@coscrad/api-interfaces';
+import { isNullOrUndefined } from '@coscrad/validation-constraints';
+import {
+    Close as CloseIcon,
+    ExpandLess as ExpandLessIcon,
+    ExpandMore as ExpandMoreIcon,
+    Hub as HubIcon,
+    TextSnippet as TextSnippetIcon,
+} from '@mui/icons-material';
+import {
+    Box,
+    Divider,
+    Drawer,
+    IconButton,
+    Stack,
+    Tooltip,
+    Typography,
+    styled,
+} from '@mui/material';
+import { ReactNode, useState } from 'react';
+import { CommandPanel } from '../commands';
+
+interface CategorizablePageLayoutProps {
+    compositeIdentifier: AggregateCompositeIdentifier;
+    selfNotesList: JSX.Element;
+    connectedResourcesList: JSX.Element;
+    commandPanel?: JSX.Element;
+    children: ReactNode;
+}
+
+const DrawerHeader = styled(Box)(({ theme }) => ({
+    display: 'flex',
+    alignItems: 'flex-start',
+    margin: theme.spacing(2, 4, 0, 4),
+    justifyContent: 'space-between',
+}));
+
+const DrawerContentStack = styled(Stack)(({ theme }) => ({
+    margin: theme.spacing(0, 4, 0, 4),
```

Comment:
Can you name these magic numbers here? I know it's annoying, but I really have no clue what these 4 numbers mean if you don't name them.